api/helpers.go

```diff
@@ -125,7 +126,9 @@ func isRedirectURLValid(config *conf.Configuration, redirectURL string) bool {
 
 	// For case when user came from mobile app or other permitted resource - redirect back
 	for _, uri := range config.URIAllowList {
-		if redirectURL == uri {
+		g := glob.MustCompile(uri)
```

Comment:
This would of course perform better if we can reuse the compiled, but not sure about the implementation (yet).

Reply:
i've moved it to compile when gotrue starts [here](https://github.com/supabase/gotrue/pull/334/files#diff-5b7f9f2790a5795a69654b2ba8c72112da865800efa1110b181c6c70de96c9dbR346-R352) and loads the config. This would allow use to reuse the compiled globs on runtime.